### DISCUSSION TITLE
SW Video mix

### DISCUSSION THREAD

**Discussion by gen4** [CESNET/UltraGrid#287] [Ideas]
Hi, I love UltraGrid and especially the SW Video mix feature.
I need to run to two UG instances with different configs for SW Video mix, however I believe that only a single place for  ~/.ug-swmix.rc is allowed. Can you add a possibility to specify an other place for a config for a second instance of UG?

Thank you for your help!

**Comment by mpiatka** [comment]
Hi, sure I've just added an option to specify the file path for the config file like this:
```
uv -t swmix:file=/tmp/example-mix.rc -t <conf> ...
```

The file path can be relative or absolute. It should be available in the newest continuous build. Let me know if this works for you.

**Reply by gen4** [reply]
You are legend! 
Works great! Thank you very much!
